manifests/generated/kv-resource.yaml

```diff
@@ -434,6 +434,28 @@ spec:
                     items:
                       type: string
                     type: array
+                  tlsConfiguration:
+                    description: TLSConfiguration holds TLS options
+                    properties:
+                      ciphers:
+                        items:
+                          type: string
+                        type: array
+                        x-kubernetes-list-type: set
+                      minTLSVersion:
+                        description: "TLSProtocolVersion is a way to specify the protocol
```

Comment:
I think the term `TLSProtocolVersion` can mislead since this name is not used in the spec. Maybe you can omit the name, just leave the description of the purpose of `minTLSVersion` field

Reply:
@enp0s3  Thanks for noticing it. I've moved the description, relating it to the `minTLSVersion` field